src/org/robockets/stronghold/robot/turntable/MenzieAlign.java

```diff
@@ -0,0 +1,92 @@
+package org.robockets.stronghold.robot.turntable;
+
+import org.robockets.stronghold.robot.Robot;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.networktables.NetworkTable;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ *
+ */
+public class MenzieAlign extends Command {
+
+	NetworkTable table;
+	boolean continuous;
+	double factor = 0.5; // It's now just an angle. SmartDashboard.getNumber("factorz", 53/1204);
+	boolean aligned = false;
+	double targetTime, startTime;
+	
+	public static final double TARGET_OFFSET = 3; // Bigger means <- left
```

Comment:
This should be common to all align commands, so when we alter the number they all alter to the same number. This will prevent cases where one align method always misses in a different way than the others.